soh/src/code/z_actor.c

```diff
@@ -3224,19 +3259,7 @@ Actor* Actor_Spawn(ActorContext* actorCtx, GlobalContext* globalCtx, s16 actorId
                                      : NULL);
     }
 
-    objBankIndex = Object_GetIndex(&globalCtx->objectCtx, actorInit->objectId);
-
-    if (objBankIndex < 0 && !gMapLoading)
-        objBankIndex = 0;
-
-    if ((objBankIndex < 0) ||
-        ((actorInit->category == ACTORCAT_ENEMY) && Flags_GetClear(globalCtx, globalCtx->roomCtx.curRoom.num))) {
-        // "No data bank!! <data bank＝%d> (profilep->bank=%d)"
-        osSyncPrintf(VT_COL(RED, WHITE) "データバンク無し！！<データバンク＝%d>(profilep->bank=%d)\n" VT_RST,
-                     objBankIndex, actorInit->objectId);
-        Actor_FreeOverlay(overlayEntry);
-        return NULL;
-    }
```

Comment:
does removing this break things?

Reply:
That's the big question. This is the part that normally is related to object dependency while the scene is loading. For enemy randomizer in specific I only need the first if statement to be gone, but the second part is what would allow Crowd Control (and possibly future mods) to also spawn in actors in a room after the clear flag is already set.

This is the part that will block this PR until we properly tested for it and have solutions to problems that arise, which I will likely need help with to figure out and will likely happen outside of this PR.

Reply:
for this pr just wrapping it in a cvar check should do the trick

Reply:
That's fair. I'll also only include the CVar in the spot that's actually needed for enemy randomizer for now as well then.